config/config.go

```diff
@@ -18,6 +19,12 @@ const (
 	// DefaultLocalRegistryPort is the default user-facing port for the local registry used for exports.
 	DefaultLocalRegistryPort = 8371
 
+	// DefaultDarwinProxyImageTag is the alpine/socat tag used for the Docker Desktop registry proxy on Darwin.
+	DefaultDarwinProxyImageTag = "1.7.4.4"
```

Comment:
I considered that but chose not to as the code requires a valid `socat` container. It's not a big concern, but perhaps it's a little less brittle to keep the image fixed. Thoughts?

Reply:
What I'm thinking is that if the author of `alpine/socat` one day decides to take this down altogether, some versions of Earthly will be just broken forever. It's good to have an escape hatch - even if the image referenced has to be of a certain specification.

Another reason is that sometimes people want to host these containers in their own registries to avoid rate limiting. They can do that only if we put the full name in the config.

We have a similar approach for this [git image](https://github.com/earthly/earthly/blob/main/config/config.go#L86) for the same reasons.

Reply:
Ok, sounds good.